src/main/java/cz/sparko/boxitory/service/FilesystemBoxRepository.java

```diff
@@ -132,4 +133,9 @@ private BoxProvider createBoxProviderFromFile(File file) {
                 hashService.getChecksum(file.getAbsolutePath())
         );
     }
+
+    private boolean containsValidBoxFile(File file) {
```

Comment:
nesla by prepouzit metoda `validateFilename`? pripadne nejak sjednotit ta logika? moc se me nelibi ze se validuje stejna vec na dvou mistech trochu ruzne, neni to moc konzistentni.